endoreg_db/serializers/video/video_file_list.py

```
@@ -0,0 +1,56 @@
+        """
+        Returns the processing status of a video as 'completed', 'in_progress', or 'available'.
+        """
+        try:
+            state = getattr(obj, 'state', None)
+            if state:
+                if state.anonymized:
+                    return 'completed'
+                elif state.frames_extracted:
+                    return 'in_progress'
+                else:
+                    return 'available'
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)